lib/ansible/modules/cloud/amazon/ec2_vpc_dhcp_options_facts.py

```diff
@@ -98,6 +98,8 @@
 
 
 def get_dhcp_options_info(dhcp_option):
+    if 'Tags' not in dhcp_option:
+        dhcp_option['Tags'] = [{'Value': '', 'Key': 'Name'}]
     dhcp_option_info = {'DhcpOptionsId': dhcp_option['DhcpOptionsId'],
                         'DhcpConfigurations': dhcp_option['DhcpConfigurations'],
                         'Tags': boto3_tag_list_to_ansible_dict(dhcp_option['Tags'])}
```

Comment:
I'm not sure. That's definitely less of a hack, but the tag Name: '' is inconsistently returned and this is to fix that. Name: '' will always exist by default. If there is another tag then Name: '' is returned as well (whereas if there is only the Name: '' no tags are returned even though it exists in the console). It just seems inconsistent that Name: '' is *sometimes* returned even though it will always exist.